content/en/serverless/step_functions/installation.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Install Serverless Monitoring for AWS Step Functions
+kind: documentation
+further_reading:
+    - link: '/serverless/configuration/'
+      tag: 'Documentation'
+      text: 'Configure Serverless Monitoring'
+    - link: "/integrations/amazon_lambda/"
+      tag: "Documentation"
+      text: "AWS Lambda Integration"
+---
+
+### Requirements
+* States must last less than 15 minutes for full traces.
+* [Associated Lambda tracing][1] is supported for Node.JS and Python runtimes.
+
+### Setup
+
+{{< tabs >}}
+{{% tab "Serverless Framework" %}}
+
+For developers using [Serverless Framework][4] to deploy their serverless applications, use the Datadog Serverless Framework Plugin.
+
+1. If you have not already, install the [Datadog Serverless Framework Plugin][1]:
+
+    ```shell
+    serverless plugin install --name serverless-plugin-datadog
+    ```
+
+2. Ensure you have deployed the [Datadog Lambda Forwarder][2], a Lambda function that ships logs from AWS to Datadog, and that you are using v3.66.0+. You may need to [update your Forwarder][5].
+
+   Take note of your Forwarder's ARN.
+
+3. Enable all logging for your Step Function. In your AWS console, open your state machine. Click *Edit* and find the Logging section. There, set *Log level* to `ALL` and enable the *Include execution data* checkbox.
```

Comment:
Should we add a screenshot here?

Reply:
<img width="618" alt="image" src="https://github.com/DataDog/documentation/assets/87997309/97794006-3a09-4cf9-b2cb-64b80d592361">